core_server/env.py

```diff
@@ -0,0 +1,31 @@
+import environ
+
+
+env = environ.Env(
+    DEBUG=(bool, False),
+    DJANGO_SECRET_KEY=str,
+    ALLOWED_HOSTS=(list, ['localhost']),
+
+    # Celery
+    CELERY_BROKER_URL=(str, "redis://redis:6379"),
+    CELERY_RESULT_BACKEND=(str, "redis://redis:6379"),
```

Comment:
I would suggest defining default values in the docker-compose file instead of here. From the docker-compose perspective, the default value makes sense but from the application, it doesn't as the services might not exist. It's better to throw errors instead. Mostly for interacting with services.

Also, in the docker-compose, we can use `?err` to make sure the user defines those values in the .env as well.
https://docs.docker.com/compose/environment-variables/
![2022-12-27-114858](https://user-images.githubusercontent.com/7059255/209619792-eb9dc569-556b-4100-9bb2-331fb2a1b615.png)

Reply:
Good point. Done.